osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -33,6 +33,8 @@ public abstract class TournamentGameBase : OsuGameBase
     {
         private const string bracket_filename = "bracket.json";
 
+        private const string stable_config = "tournament/stable.json";
```

Comment:
Shouldn't be defined four times (here, in `FileBasedIPC`, in `StablePathSelectScreen` and in `SetupScreen`).